### PR TITLE
Use channel map in TPG

### PR DESCRIPTION
**Pull request by philiprodrigues**
This PR adds use of the channel map to the software TPG. It's a manual forward-port of the same changes that were made on the 2.8.2 readout release for the VD coldbox tests in December. This PR depends on https://github.com/DUNE-DAQ/readoutlibs/pull/6 . So far, it builds but isn't tested. Before I do some serious testing, I'd like to get a check that I've put the code in the right place in the new readout-package structure.

### CODE REVIEW

File: include/fdreadoutlibs/wib/tpg/RegisterToChannelNumber.hpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/**
+ * @file RegisterToChannelNumber.hpp Convert from WIB1 data AVX register position to offline channel numbers
+ *
+ * This is part of the DUNE DAQ , copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef READOUT_SRC_WIB_TPG_REGISTERTOCHANNELNUMBER_HPP_
```
Comment (roland-sipos) [inline include/fdreadoutlibs/wib/tpg/RegisterToChannelNumber.hpp:9]: Please update the include guard.

**Review by roland-sipos — CHANGES_REQUESTED** [review]

**Comment by roland-sipos** [comment]
Also, could you please have a look on why the CI fails?

`/__w/fdreadoutlibs/fdreadoutlibs/dev-c7/sourcecode/fdreadoutlibs/include/fdreadoutlibs/wib/WIBFrameProcessor.hpp:226:64: error: ‘struct dunedaq::readoutlibs::readoutconfig::RawDataProcessorConf’ has no member named ‘channel_map_name’; did you mean ‘channel_map_rce’?`

**Comment by philiprodrigues** [comment]
I think the CI failure is because the CI doesn't have https://github.com/DUNE-DAQ/readoutlibs/pull/6 merged, which is what provides new variable that's needed. I'm not sure of the best way to deal with that

**Comment by roland-sipos** [comment]
Hi @philiprodrigues ! Indeed this was not merged in yet, doing it right now.

**Review by roland-sipos — APPROVED** [review]

**Pull request merged**